main.go

```diff
@@ -732,9 +734,37 @@ func executeLinter(state *linterState, args []string) error {
 			state.Name)
 	}
 
-	processOutput(state, buf.Bytes())
+	cmdOutput, err := preprocessLinterPanic(state.Name, buf, err)
+	processOutput(state, cmdOutput)
 	debug("%s linter took %s", state.Name, time.Since(start))
-	return nil
+	return err
+}
+
+const panicExitStatus = 2
+
+func preprocessLinterPanic(name string, buf *bytes.Buffer, processError error) ([]byte, error) {
+	if exitStatus(processError) == panicExitStatus {
+		var err error
+		cmdOutput := new(bytes.Buffer)
+		routines, _ := stack.ParseDump(buf, cmdOutput)
```

Comment:
@bradleyfalzon  Yes, great idea! This is much better, I'll make this change now.

@pierrre As was said, errors here are not significant. There are plenty of reasons for a process to exit with status 2, a failure to parse just means we don't have a stack trace.